poucave/utils.py

```diff
@@ -148,7 +148,14 @@ def render_checks(func):
     async def wrapper(request):
         # First, check that client requests supported output format.
         is_text_output = False
-        accepts = set(request.headers.getall("Accept", []))
+        # Several Accept headers can be specified, and each of them can have
+        # comma-separated values (eg. `Accept: text/html,application/xhtml+xml`).
```

Comment:
The full format is [rather complex](https://tools.ietf.org/html/rfc7231#section-5.3.2). Can we use a library to parse the header? While we don't have to support all formats clients could possibly send, using a lib would reduce the likelihood we have to fix more bugs in this code – it's starting to become non-trivial already.

I know that there is `parse_accept_header` in Werkzeug, which is what I used to use when working with Flask. What do you think?